engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/compile/CustomNodeValidationSpec.scala

```diff
@@ -226,7 +226,7 @@ class CustomNodeValidationSpec extends FunSuite with Matchers with OptionValues
     }
   }
 
-  test("valid process using context transformation api - union") {
+  test("valid process using context transformation api with global variables - union") {
```

Comment:
The rule of thumb should be: new bug - new test but ... we I can close eye to this ;-)